src/arr.ts

```diff
@@ -109,6 +173,52 @@ export class Arr<T> {
     return this.values.length
   }
 
+  /**
+   * Returns the max value in the array.
+   *
+   * @returns {Number}
+   */
+  max (): number {
+    return Math.max(...this.values.map((value: T) => +value))
+  }
+
+  /**
+   * Returns median of the current array.
+   *
+   * @param {}
+   *
+   * @returns {Number}
+   */
+  median (): number {
+    const sorted: Arr<T> = this.sort((a: any, b: any) => a - b)
+
+    const mid: number = Math.floor(this.size() / 2)
+
+    return this.size() % 2 !== 0
+      ? +sorted.all()[mid]
+      : (+sorted.all()[mid] + +sorted.all()[(mid - 1)]) / 2 // eslint-disable-line
+  }
+
+  /**
+   * Returns the min value in the array.
+   *
+   * @returns {Number}
+   */
+  min (): number {
+    return Math.min(...this.values.map((value: T) => +value))
+  }
+
+  /**
+   * Removes and returns the last item from the array.
+   *
+   * @param {}
+   *
+   * @returns {Number}
+   */
+  pop (): any {

```

Comment:
```suggestion
  pop (): T {

```

Reply:
@marcuspoehls  thanks for review, i'll fix it asap, sorry for late response.

Reply:
@mcvladthegoat You’re welcome 😃  no problem!